src/CommunityToolkit.Maui.Core/Essentials/FolderPicker/IFolderPicker.shared.cs

```diff
@@ -11,12 +13,22 @@ public interface IFolderPicker
 	/// <param name="initialPath">Initial path</param>
 	/// <param name="cancellationToken"><see cref="CancellationToken"/></param>
 	/// <returns><see cref="FolderPickerResult"/></returns>
+	[SupportedOSPlatform("Android26.0")]
+	[SupportedOSPlatform("iOS14.0")]
+	[SupportedOSPlatform("MacCatalyst14.0")]
+	[SupportedOSPlatform("Tizen")]
+	[SupportedOSPlatform("Windows")]
 	Task<FolderPickerResult> PickAsync(string initialPath, CancellationToken cancellationToken);
 
 	/// <summary>
 	/// Allows the user to pick a folder from the file system
 	/// </summary>
 	/// <param name="cancellationToken"><see cref="CancellationToken"/></param>
 	/// <returns><see cref="FolderPickerResult"/></returns>
+	[SupportedOSPlatform("Android")]
+	[SupportedOSPlatform("iOS14.0")]
+	[SupportedOSPlatform("MacCatalyst14.0")]
+	[SupportedOSPlatform("Tizen")]
+	[SupportedOSPlatform("Windows")]
```

Comment:
I'm not aware of any problems with this feature on iOS 13 and lower? Does this feature not work on iOS 13, for example? 

If there is a known-issue on iOS 13, let's open an Issue and comment here with the link. 

If not, we can remove all of these attributes.

```suggestion
```